Sources/FoundationEssentials/Data/DataProtocol.swift

```diff
@@ -10,11 +10,11 @@
 //
 //===----------------------------------------------------------------------===//
 
-#if os(macOS) || os(iOS)
+#if canImport(Darwin)
 import Darwin
-#elseif os(Linux)
+#elseif canImport(Glibc)
```

Comment:
@MaxDesiatov Do we need a `canImport` for musl here as well?

Reply:
Yes, of course, thanks for catching this!